src/main/java/com/pnudev/springonlinedelivery/Menu_Item.java

```diff
@@ -0,0 +1,24 @@
+package com.pnudev.springonlinedelivery;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Table
+@Entity
+public class Menu_Item {
```

Comment:
Fix class naming (refer to https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html for more info)

Reply:
Fixed